index.js

```diff
@@ -123,62 +103,23 @@ function YahooFantasy(consumerKey, consumerSecret) {
   // this.users.fetch = _.bind(this.users.fetch, this);
 }
 
-YahooFantasy.prototype.setUserToken = function(userToken, userSecret, userSession) {
-  this.yuser.token = userToken;
-  this.yuser.secret = userSecret;
-  this.yuser.sessionHandle = userSession;
-};
-
-YahooFantasy.prototype.refreshUserToken = function() {
-  var deferred = Q.defer();
-  var self = this;
-
-  var now = Math.floor(new Date().getTime() / 1000);
-  var refresh_data = querystring.stringify({
-    oauth_nonce: now.toString(36),
-    oauth_consumer_key: self.consumer.key,
-    oauth_signature_method: 'plaintext',
-    oauth_signature: self.consumer.secret + '&' + self.yuser.secret,
-    oauth_version: '1.0',
-    oauth_token: self.yuser.token,
-    oauth_timestamp: now,
-    oauth_session_handle: self.yuser.sessionHandle
-  });
-
-  https.get('https://api.login.yahoo.com/oauth/v2/get_token?' + refresh_data, function(res) {
```

Comment:
To exchange the refresh token in OAuth 2.0, we need a redirectionUri and must handle Yahoo's response in it. So external app must handle the refresh token like below.
1. [externalApp] Call some yfsapi API.
2. [yfsapi] If session is expired, we return an error.
4. [externalApp] [Exchange the refresh token for new access token](https://developer.yahoo.com/oauth2/guide/flows_authcode/#refresh-token-label=).
5. [externalApp] Call yf.setUserToken(NEW USER_TOKEN) API.
6. [externalApp] Call some yfsapi API.